Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -775,7 +866,9 @@ int SVertexer::checkCascades(const V0Index& v0Idx, const V0& v0, float rv0, std:
       continue; // skip the track used by V0
     }
     auto& bach = tracks[it];
-
+    if (mSVParams->mSkipTPCOnlyCascade && bach.hasTPC) {
```

Comment:
This way you skip bachelors with TPC not with TPConly. One should use `if (mSVParams->mSkipTPCOnlyCascade && bach.nITSclu<1)`